### PR TITLE
Updating readmes

### PR DESCRIPTION
**Pull request by ivannaranjo**
Updating the readmes with some new information learnt from the work on this relase.

### CODE REVIEW

File: GoogleCloudExtension/ProjectTemplates/README.md [hunk 1]
```diff
@@ -12,5 +12,20 @@ The typical set of steps then is:
 * Use the experimental Visual Studio instance to create a new project based on your changed template, verify that the template expands to what you expect. Verify that the project produced from the template builds.
 * Repeat.
 
+## To add package references to an existing template
+The procedure to add a new package reference to an existing template differs between an ASP.NET Core 1.0 app and an ASP.NET 4.x app.
+
+### Adding a package reference to an ASP.NET 4.x app
+Adding a package reference to an ASP.NET 4.x aproject is a very involved process because you need to reproduce what Visual Studio normally does when adding the package.
```
Comment (iantalarico) [inline GoogleCloudExtension/ProjectTemplates/README.md]: aproject -> project

File: GoogleCloudExtension/ProjectTemplates/README.md [hunk 2]
```diff
@@ -12,5 +12,20 @@ The typical set of steps then is:
 * Use the experimental Visual Studio instance to create a new project based on your changed template, verify that the template expands to what you expect. Verify that the project produced from the template builds.
 * Repeat.
 
+## To add package references to an existing template
+The procedure to add a new package reference to an existing template differs between an ASP.NET Core 1.0 app and an ASP.NET 4.x app.
+
+### Adding a package reference to an ASP.NET 4.x app
+Adding a package reference to an ASP.NET 4.x aproject is a very involved process because you need to reproduce what Visual Studio normally does when adding the package. 
+
+The best way to know what chages are needed is to create a new project from the template and, using Visual Studio, add the package. Doing a diff with the unchanged project will show you of the changes you need to port to the template. These changes usually involve:
```
Comment (iantalarico) [inline GoogleCloudExtension/ProjectTemplates/README.md]: chages -> changes

File: README.md [hunk 3]
```diff
@@ -1,30 +1,28 @@
...
-the `reset-windows-password` command to be installed. To install this command
-install the `beta` command group as follows:
+## Dependencies on Google Cloud SDK
+The Visual Studio extension depends on the Cloud SDK for certain functionality, make sure that you have at least version 146.0.0 of the Cloud SDK installed when running the extension.
+
+You can install the Google Cloud SDK from <https://cloud.google.com/sdk/>. You will also need to make sure that you have the "beta" and "kubectl" components.
+* The "beta" component is used for Windows VMs password management and for the deployment ASP.NET Core apps to App Engine Flex.
+* The "kubectl" component is used for interactions with Google Container Engine.
+
+You can make sure that you have all the necessary components installed by running the cokmmand:
```
Comment (iantalarico) [inline README.md]: cokmmand -> command

File: GoogleCloudExtension/ProjectTemplates/README.md (hunk 1 above)
Comment (ivannaranjo) [inline GoogleCloudExtension/ProjectTemplates/README.md]: done.

File: GoogleCloudExtension/ProjectTemplates/README.md (hunk 2 above)
Comment (ivannaranjo) [inline GoogleCloudExtension/ProjectTemplates/README.md]: Done.

**Pull request merged**